### DISCUSSION TITLE
Unable to synthesize swerv_wrapper on ASAP7

### DISCUSSION THREAD

**Discussion by gudeh** [The-OpenROAD-Project/OpenROAD-flow-scripts#1582] [General]
Hi everyone. When I try to synthesize swerv_wrapper as available in the Makefile: "DESIGN_CONFIG=./designs/asap7/swerv_wrapper/config.mk"

Is this design supposed to run successfully? The process was killed with "code 9" on step: 5_3_route

**Comment by gudeh** [comment]
[5_3_route.tmp.log](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/files/13079057/5_3_route.tmp.log)

**Comment by maliberty** [comment]
Most likely you ran out of memory.  How much ram do you have?

**Reply by gudeh** [reply]
I have 16GB on this PC. Should it be enough?

**Reply by maliberty** [reply]
How much swap do you have?

**Reply by gudeh** [reply]
```
gudeh@snowflake:~$ free --giga
               total        used        free      shared  buff/cache   available
Mem:              16           2          10           0           2          12
Swap:              2           0           2

```

**Comment by maliberty** [comment]
drt uses (https://jenkins.openroad.tools/blue/rest/organizations/jenkins/pipelines/OpenROAD-flow-scripts-Nightly-Public/runs/3170/nodes/735/log/?start=0)
```
Elapsed time: 1:16:29[h:]min:sec. CPU time: user 72021.27 sys 1097.28 (1593%). Peak memory: 24364784KB.
```
which won't fit in 16 Gb + 2 GB of swap.
